common/amundsen_common/models/table.py

```diff
@@ -183,6 +183,7 @@ class Table:
     owners: List[User] = []
     watermarks: List[Watermark] = []
     table_writer: Optional[Application] = None
+    table_apps: Optional[List[Application]] = None
```

Comment:
is it for backward compatible that we keep both table_writer and table_apps ? should we just change table_writer to `Optional[List[Application]]` type?

Reply:
yes. There is no way to repurpose the field without doing something much uglier (e.g. custom serializer in marshmallow). I looked hard.